extension/app/ts/utils/wire-types.ts

```diff
@@ -801,6 +801,12 @@ export const GetSimulationStack = t.ReadonlyArray(t.Intersect(
 	}).asReadonly(),
 ))
 
+export type GetSimulationStack = t.Static<typeof GetSimulationStack>
+export const GetSimulationStack = t.Object({
+	method: t.Literal('interceptor_getSimulationStack'),
+	params: t.Tuple(t.Union(t.Literal('1.0.0'))),
```

Comment:
You may have done this intentionally since we will *eventually* add other versions, in which case feel free to ignore this comment.
```suggestion
	params: t.Tuple(t.Literal('1.0.0')),
```

Reply:
Haha, it was directly your suggestion from the previous version :). But yeah, we can remove it was its bit premature

Reply:
I can totally believe that.  🤣